blocks/kanban-board/src/components/board-title/board-title.tsx

```diff
@@ -0,0 +1,85 @@
+import clsx from "clsx";
```

Comment:
Yes, please. Let's do that.

Reply:
Same for the text inside cards?

![Uploading editable.gif…]()

Reply:
Yes, please!

Reply:
Created a [task](https://app.asana.com/0/1204000740778938/1204256944614937/f) _(internal)_ for this, since it needs a bit fine-tuning with the styling of the `EditableField`.